.github/interop/required.json

```diff
@@ -233,4 +379,4 @@
     "quiche": [],
     "xquic": []
   }
-}
+}
```

Comment:
did you mean to remove the trailing newline?

Reply:
i just let the [autoformatter](https://prettier.io/) do its thing